etl/steps/data/meadow/irena/2024-10-29/renewable_power_generation_costs.py

```diff
@@ -191,35 +230,70 @@ def extract_country_cost_from_excel_file(data: pr.ExcelFile) -> Table:
     # Extract LCOE for specific countries and technologies (those that are available in original data).
 
     # Solar photovoltaic.
+    # NOTE: For some reason, sheet 3.11 contains LCOE from 2010 to 2023 for 15 countries, and 3.12 contains LCOE from 2018 to 2023 for 19 countries.
```

Comment:
Have you re-checked why is this the case? Maybe it's worth asking them

Reply:
I don't think there's any particular reason. I suppose they simply preferred to show certain countries in one figure, and other countries in the other figure. It would be good to have access to underlaying combined data (which I suppose lives somewhere), but I don't think that's currently possible.